tests/Functional/SelfCommandsTest.php

```diff
@@ -27,7 +27,6 @@ public function testSelfUpdateCommand()
             0 !== $exitCode
             && false !== strpos($error, 'rate limit exceeded')
         ) {
-            // @todo: fix CMS-972
```

Comment:
CMS-972 == LOPS-972, i.e. the ticket this PR is for.  Are we removing this comment because it was a reminder to do the work we're working on right now?